README.md

```diff
@@ -53,10 +53,10 @@ We are currently defining the Initiatives for 2023, [feel free to participate](h
 
 | Initiative           | Champion                                         | Status                                   | Links
 |----------------------|--------------------------------------------------|------------------------------------------|-------------------------------------------------
-| Permission Model | [@RafaelGSS](https://github.com/RafaelGSS) |  PR In Review | [PR #44004](https://github.com/nodejs/node/pull/44004)
+| Permission Model | [@RafaelGSS](https://github.com/RafaelGSS) |  In Progress | [PR #44004](https://github.com/nodejs/node/pull/44004)
 | Automate update dependencies | [@facutuesca](https://github.com/facutuesca) | In Progress | [Issue #828](https://github.com/nodejs/security-wg/issues/828)
-| Flag to emit a warning when vulnerable | [@RafaelGSS](https://github.com/RafaelGSS)  | Evaluation | _pending_
-
+| Assessment against best practices | [@fraxken](https://github.com/fraxken) | Evaluation | TBD
```

Comment:
@fraxken can you create an issue to keep track of it? You can also link the OSSF Scorecard as one actionable

Reply:
Done: https://github.com/nodejs/security-wg/issues/859